native_client/wasm/test.html

```diff
@@ -5,9 +5,17 @@
     <title>WASM Sample</title>
   </head>
   <body>
+    <script src="audiobuffer-to-wav/index.js"></script>
```

Comment:
@dsouza95 would you feel comfortable/be ok with making this a separate file, e.g. `test_live.html`? I believe this would make it easier for other folks to understand. What do you think?

Reply:
Sure thing, it will be clearer indeed :)